src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -80,6 +85,21 @@ class TestChannel(AioTestBase):
             self.assertIsNotNone(
                 exception_context.exception.trailing_metadata())
 
+    async def test_unary_call_metadata(self):
+        async with aio.insecure_channel(self._server_target) as channel:
+            hi = channel.unary_unary(
+                _UNARY_CALL_METHOD,
+                request_serializer=messages_pb2.SimpleRequest.SerializeToString,
+                response_deserializer=messages_pb2.SimpleResponse.FromString)
+            call = hi(messages_pb2.SimpleRequest(), metadata=_INVOCATION_METADATA)
+            initial_metadata = await call.initial_metadata()
+            trailing_metadata = await call.trailing_metadata()
+
+            self.assertIsInstance(initial_metadata, tuple)
+            self.assertEqual(_INVOCATION_METADATA[0], initial_metadata[0])
+            self.assertIsInstance(trailing_metadata, tuple)
+            self.assertEqual(_INVOCATION_METADATA[1], trailing_metadata[0])
```

Comment:
I think this function should be moved to `call_test.py` since it is testing member methods of `Call` object.